fiboa_cli/datasets/jp.py

```diff
@@ -0,0 +1,74 @@
+from ..convert_utils import convert as convert_
+import pandas as pd
+
+YEARS = [
+    f"https://data.source.coop/pacificspatial/field-polygon-jp/parquet/jp_field_polygons_{year}.parquet"
+    for year in range(2021, 2024+1)]
+SOURCES = YEARS[-1]
```

Comment:
Similarly here, why are there to variables? The whole flow of generating the sources looks confusing to me without further context.